pkg/spec/backup.go

```diff
@@ -93,6 +93,11 @@ type S3Source struct {
 	// S3Bucket overwrites the default etcd operator wide bucket.
 	S3Bucket string `json:"s3Bucket,omitempty"`
 
+	// Prefix is the S3 prefix used to prefix the bucket path.
+	// It's the prefix at the beginning.
+	// After that, it will have version and cluster specific prefix appended.
```

Comment:
have version and cluster specific prefix appended -> have version and cluster specific paths